dassl/utils/torchtools.py

```diff
@@ -251,16 +251,25 @@ def open_specified_layers(model, open_layers):
                 p.requires_grad = False
 
 
-def count_num_param(model):
+def count_num_param(model=None, params=None):
     r"""Count number of parameters in a model.
 
     Args:
         model (nn.Module): network model.
-
+        params: network model`s params.
     Examples::
         >>> model_size = count_num_param(model)
     """
-    return sum(p.numel() for p in model.parameters())
+    if model:
```

Comment:
Use `if model is not None`, which is more informative.

Also add space between blocks to make the code more readable.